src/lib/EventPublisher.js

```diff
@@ -156,11 +164,20 @@ class EventPublisher extends EventEmitter {
         },
         subscriptionTimeout,
       );
-
-      this.once(eventNamePrefix, (): void => clearTimeout(timeout));
+      this.once(eventNamePrefix, () => {
+        cleared = true;
+        clearTimeout(timeout);
+      });
     }
 
     if (once) {
+      if (onceUnsubscribe === true) {
```

Comment:
In case of a timeout unsubscribe is called, so a Second call to unsubscribe will throw an error. So we have to make sure that unsubscribe is only called once.